utils/nr-graphql-helpers.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+const fetch = require('node-fetch');
+
+/**
+ * Build body param for NR GraphQL request
+ * @param {{queryString, variables}} queryBody - query string and corresponding variables for request
+ * @returns {String} returns the body for the request as string
+ */
+const buildRequestBody = ({ queryString, variables }) =>
+  JSON.stringify({
+    ...(queryString && { query: queryString }),
+    ...(variables && { variables }),
+  });
+
+/**
+ * Send NR GraphQL request
+ * @param {{queryString, variables}} queryBody - query string and corresponding variables for request
+ * @param {String} url - request URL
+ * @param {String} token - API token for request
+ * @returns {Object} An object with the results or errors of a GraphQL request
+ */
+const fetchNRGraphqlResults = async (queryBody, url, token) => {
```

Comment:
I like what Cayla did on her PR, accessing the `env` values directly rather than passing them in. Let's do that here

```
const NR_API_URL = process.env.NR_API_URL;
const NR_API_TOKEN = process.env.NR_API_TOKEN;
```